Packs/cyberark_AIM/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -0,0 +1,231 @@
+            "GET",
+            url_suffix='/PasswordVault/api/Accounts',
+            resp_type="json",
+            headers=headers,
+            params={
+                'offset': offset,
+                'limit': limit
+            }
+        )
+        return res
+
+    def add_account(self, address, user_name, platform_id, safe_name,
+                    name='', secret='', secret_type='password', platform_account_properties='',
+                    automatic_management_enabled='true', manual_management_reason='', remote_machines='',
+                    access_restricted_to_remote_machines='false'):
+        LOG('Adding a new Account')
+
+        headers = {
+            'Authorization': self.auth_token
+        }
+
+        body = {
+            "userName": user_name,
+            "address": address,
+            "platformId": platform_id,
+            "safeName": safe_name
+        }
+
+        if name:
+            body['name'] = name
+
+        if secret and secret_type:
+            body['secret'] = secret
+            body['secretType'] = secret_type
+
+        if platform_account_properties:
+            body['platformAccountProperties'] = platform_account_properties
+
+        if automatic_management_enabled == "false":
+            body['secretManagement'] = {
+                "automaticManagementEnabled": automatic_management_enabled,
+                "manualManagementReason": manual_management_reason
+            }
+
+        if remote_machines:
+            body['remoteMachinesAccess'] = {
+                "remoteMachines": remote_machines,
+                "accessRestrictedToRemoteMachines": access_restricted_to_remote_machines
+            }
+        res = self._http_request(
+            "POST",
+            url_suffix='/PasswordVault/api/Accounts',
+            resp_type="json",
+            headers=headers,
+            data=body
+        )
+        return res
+
+
+'''' Commands '''
+
+
+def test_module(client):
+    client.get_system_summary()
+    return 'ok'
+
+
+def list_accounts(client, args):
+    title = f'{INTEGRATION_NAME} - List of the Accounts'
+    raws = []
+    cyberark_ec = []
+    raw_response = client.get_accounts(offset=args['offset'], limit=args['limit'])['value']
+
+    if raw_response:
+        for item in raw_response:
+            raws.append(item)
+            cyberark_ec.append({
+                'AccountName': item['name'],
+                'UserName': item['userName'],
+                'PlatformID': item['platformId'],
+                'SafeName': item['safeName'],
+                'AccountID': item['id'],
+                'CreatedTime': item['createdTime']
+            })
+
+    if not raws:
+        return f'{INTEGRATION_NAME} - Could not find any Accounts'
+
+    context_entry = {
+        "CyberArk.Accounts": cyberark_ec
+    }
+
+    human_readable = tableToMarkdown(t=context_entry.get('CyberArk.Accounts'), name=title)
+    return [human_readable, context_entry, raws]
```

Comment:
Maybe we should update this to use `CommandResults` and `return_results` instead of the existing `return_outputs` method.
You can find information about how to do this here: https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults and we can also discuss about it in our demo.